ts/utils/timer.ts

```diff
@@ -0,0 +1,8 @@
+import BackgroundTimer from "react-native-background-timer";
+
+export function startTimer(t: number): Promise<never> {
+  // tslint:disable-next-line:promise-must-complete
```

Comment:
why tslint was complaining here?

Reply:
It seem related to https://github.com/Microsoft/tslint-microsoft-contrib/issues/329